src/screens/DiscoveryScreenV2/fragment/SearchFragment.js

```diff
@@ -1,20 +1,22 @@
 import * as React from 'react';
-import { ScrollView, StyleSheet } from 'react-native';
+import {ScrollView, StyleSheet} from 'react-native';
 
 import DiscoveryTitleSeparator from '../elements/DiscoveryTitleSeparator';
 import RecentSearch from '../elements/RecentSearch';
-import { colors } from '../../../utils/colors';
+import {COLORS} from '../../../utils/theme';
 
-const SearchFragment = () => <ScrollView style={styles.container} keyboardShouldPersistTaps={'handled'}>
-        <RecentSearch />
-        <DiscoveryTitleSeparator text='Suggested Users' /> 
-    </ScrollView>;
+const SearchFragment = () => (
+  <ScrollView style={styles.container} keyboardShouldPersistTaps={'handled'}>
+    <RecentSearch />
+    <DiscoveryTitleSeparator text="Suggested Users" />
+  </ScrollView>
+);
 
 const styles = StyleSheet.create({
-    container: {
-        flex: 1,
-        backgroundColor: colors.white
-    }
-})
+  container: {
+    flex: 1,
+    backgroundColor: COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color property has been updated to use `COLORS.white` instead of `colors.white`. This is a good change as it ensures consistency with the new import statement.

```diff
- backgroundColor: colors.white
+ backgroundColor: COLORS.white
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->